library/zitilib.c

```diff
@@ -1171,9 +1171,9 @@ static void do_enroll(ziti_enroll_opts *opts, future_t *f, uv_loop_t *loop) {
 
 int Ziti_enroll_identity(const char *jwt, const char *key, const char *cert, char **id_json, unsigned long *id_json_len) {
     ziti_enroll_opts opts = {
-            .jwt_content = jwt,
-            .enroll_key = key,
-            .enroll_cert = cert,
+            .token = jwt,
```

Comment:
why rename these and why would this only be a minor bump? shouldn't this be a major bump since the api is changing?

Reply:
`enroll_` prefix is redundant. it is minor to be consistent with how we always versioned things